test/api/v3/integration/tasks/POST-tasks_id_score_direction.test.js

```diff
@@ -208,6 +208,13 @@ describe('POST /tasks/:id/score/:direction', () => {
       expect(task.completed).to.equal(false);
     });
 
+    it('computes isDue', async () => {
+      await user.post(`/tasks/${daily._id}/score/up`);
+      let task = await user.get(`/tasks/${daily._id}`);
+
+      expect(task.isDue).to.equal(true);
```

Comment:
isDue has nothing to do with completed as it only represents shouldDo. isDue is combined with a completed check on the UI.